inc/class-rewrite.php

```diff
@@ -57,11 +57,17 @@ public function flush() {
 	/**
 	 * Override the category link to remove the category base.
 	 *
-	 * @param string $link Unused, overridden by the function.
+	 * @param string  $link     Unused, overridden by the function.
```

Comment:
This is no longer true, because it _is_ returned when `$taxonomy` is not `category`.